Lib/pathlib.py

```diff
@@ -226,6 +223,7 @@ def is_reserved(self, parts):
 
     def make_uri(self, path):
         # Under Windows, file URIs use the UTF-8 encoding.
+        from urllib.parse import quote_from_bytes as urlquote_from_bytes
```

Comment:
What is the reason of the renaming?

Reply:
there is no reason, you are right but this is just a copy. Now he could remove the renaming.